.env.example

```diff
@@ -1,17 +1,26 @@
 ########### General settings with defaults ###########
-ENTRY_POINT=0x5FF137D4b0FDCD49DcA30c7CF57E578a026d2789
+ENTRY_POINT=0x0000000071727De22E5E9d8BAf0edAc6f37da032
 VERIFYING_SIGNER=0xf...6
 PAYMASTER_OWNER=0x7...C
 RPC_URL=http://127.0.0.1:8545
 
 
 ########### Wallet info & secrets ###########
 MSCA_FACTORY_OWNER_PRIVATE_KEY=0x9...2
+MSCA_FACTORY_OWNER_ADDRESS=0x0...5
 DEPLOYER_PRIVATE_KEY=0x8...6
 DEPLOYER_ADDRESS=0xB...E
 
-
 ########### API keys ###########
 INFURA_API_KEY=
 ALCHEMY_API_KEY=
 ETHERSCAN_API_KEY=
+
+########### Contract addresses ###########
```

Comment:
Consider moving these to a Solidity file?
We will have more addresses as the product iterates